src/widget-core/WidgetBase.ts

```diff
@@ -52,15 +52,38 @@ function isDomMeta(meta: any): meta is Base {
 	return Boolean(meta.afterRender);
 }
 
-const IGNORE_LIST: (string | symbol)[] = ['constructor', 'render'];
+const IGNORE_LIST: (string | symbol)[] = ['render', ...Object.getOwnPropertyNames(Object.getPrototypeOf({}))];
```

Comment:
We can exclude all the properties that come from `Object.prototype`. Technically we might not even need this since we break our loop on subclasses of `WidgetBase`?